docs/source/examples.rst

```diff
@@ -35,6 +35,7 @@ GLM
 ===
 
 .. toctree::
+   notebooks/RatTumorExample.ipynb
```

Comment:
Could you please move this to the last session of GLM, also change the title of the notebook:
`rat tumor`--> `Hierarchical Binominal Model: Rat Tumor Example`